python/lsst/ts/lasertracker/laser_tracker_csc.py

```diff
@@ -667,6 +645,32 @@ async def measure_alignment(self, target: str) -> None:
             target_offset["dRX"] -= self.config.zero_points["camera"]["u"]
             target_offset["dRY"] -= self.config.zero_points["camera"]["v"]
 
+        # Add LUT
+
+        # compensation class for CAM LUT offsets
+        self.compensation = compensation.Compensation(
+            elevation_rotation_coeffs=self.config.targets_lut_coeffs["CAM"]["elevation_rotation_coeffs"],
+            azimuth_coeffs=self.config.targets_lut_coeffs["CAM"]["azimuth_coeffs"],
+            temperature_coeffs=self.config.targets_lut_coeffs["CAM"]["temperature_coeffs"],
+            min_temperature=-20.0,
```

Comment:
These should probably not be hard coded like this and instead be defined in the configuration.